packages/playground-ui/src/ds/components/TokenBudget/token-budget.css

```
@@ -0,0 +1,66 @@
+/* Annulus is the track, wedge is the fill, and while work runs a highlight
+ * travels the wedge — `shimmer-text` transposed to an arc, so nothing moves
+ * beside the dial and nothing reads as a rendering glitch. */
+/* Hatching reads as provisional: the slice of the fill a pending pass will free. */
```

Stylelint (17.14.0)

[error] Expected empty line before comment (comment-empty-line-before)

(comment-empty-line-before)